tests/wake-format/basic/stdout

```diff
@@ -1038,4 +942,3 @@ export def makeJSONRunner (plan: JSONRunnerPlan): Runner =
 
     json
 
-
```

Comment:
There's still an unnecessary blank line; GitHub won't show the EOF character (unless it's missing entirely and gets the red warning symbol), so any that show up in review are extra.

Reply:
Ah rip, that is what I get for trusting the diff lol

Reply:
Okay, actually I'm confused. This is correct no? We want 1 empty line right?

The file above has the following content
```
...cut....
    json<newline>
<newline><eof>
```

Reply:
We'd want `json\n` where where the single newline is the EOF character.  The original output had `json\n\n\n` and the change cuts it down to `json\n\n` but there's still an extra line in there.  GitHub hides the very last newline, but if it's showing something numbered then there's still an extra "empty string" being tokenized.

Reply:
I supposed I'm confused by why `json\n\n<eof>` is wrong. That feels like a pretty standard thing to do. (`json\n\n\n<eof>` is clearly wrong)

Reply:
Ah, I see where you're coming from then.  It's not *morally* wrong if that's your workflow, no, but it's not the style we've been using; all of the existing files in the repo I'm aware of don't do that, we've pointed it out in the past as unnecessary in code reviews as a style recommendation, and with leaving it as-is, every Wake file wherever the formatter is run will be touched to add space there where there (with potentially very few exceptions in the most obscure corners) wasn't any space previously.